project.yaml

```diff
@@ -77,4 +77,13 @@ actions:
     needs: [data_development_pf_code_distinct, data_development_pf_code_events, data_development_pf_codes_pathways]
     outputs:
       moderately_sensitive:
-        dataset: output/data_development/pf_codes_count_summary.csv
\ No newline at end of file
+        dataset: output/data_development/pf_codes_count_summary.csv
+
+  generate_non_pf_medications_measures:
+    run: >
+      ehrql:v1 generate-measures analysis/measures_definition_nonpf_med_counts.py
+      --dummy-tables dummy_tables
+      --output output/measures/nonpf_medications_measures.csv
+    outputs:
+      moderately_sensitive:
+        measure: output/measures/nonpf_medications_measures.csv
```

Comment:
We should find a better file name now that we are counting pf and non pf medications